astroquery/alma/core.py

```diff
@@ -1055,6 +1055,21 @@ def _json_summary_to_table(self, data, base_url):
         tbl = Table([Column(name=k, data=v) for k, v in iteritems(columns)])
         return tbl
 
+    def get_project_metadata(self, projectid, cache=True):
+        """
+        Get the metadata - specifically, the project abstract - for a given project ID.
+        """
+        url = urljoin(self._get_dataarchive_url(), 'aq/')
+
+        assert len(projectid) == 14, "Wrong length for project ID"
```

Comment:
I'm not particularly excited about these asserts in code, but I suppose it'll do for now and we can clean up later (famous last words).